Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.10-stretch
+ADD . /go/src/github.com/rqlite/rqlite
+WORKDIR /go/src/github.com/rqlite/rqlite
+RUN go get ./...
+RUN go install -ldflags="-X main.commit=$(git rev-parse HEAD) -X main.branch=$(git rev-parse --abbrev-ref HEAD)" ./cmd/...
+
+FROM frolvlad/alpine-glibc:alpine-3.8
```

Comment:
How do I know I can trust this container?

Reply:
I am assuming that this image forms the basis of the rqlite image. Let me know if I am missing something.

Reply:
yes that would be the base of your image. Its an automated build you can see the dockerfile here: https://hub.docker.com/r/frolvlad/alpine-glibc/~/dockerfile/

The reason behind this base image is, that I wasn't able to get a fully statically compiled binary version of rqlite. What I tried was the usual `CGO_ENABLED=0` but than I got some other error messages. I guess some of your dependencies does need cgo...

An alternative is that you start from an official alpine, and add the glibc binary packages (they are not officially released) so its 4 lines:
```
apk --no-cache add ca-certificates wget
wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub
wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.27-r0/glibc-2.27-r0.apk
apk add glibc-2.27-r0.apk
```

Reply:
OK, thanks @lalyos for the extra information. Need to do a little bit more research. If you can provide any references to this glibc being used by others, that would be helpful.